src/main/kotlin/org/rust/ide/utils/ExprUtils.kt

```diff
@@ -195,3 +200,30 @@ fun RsExpr.evalBooleanExpression(): Boolean? {
 }
 
 private fun createPsiElement(project: Project, value: Any) = RsPsiFactory(project).createExpression(value.toString())
+
+/***
+ * Go to the item above RsParenExpr.
+ *
+ * @return a parent expression without parentheses.
+ */
+fun RsExpr.skipParenExprUp(): PsiElement? {
```

Comment:
I think, it would be better for user of this function if it would return `RsExpr`, which parent is not `RsParenExpr`. Now usage of this function is complicated, because you should check return type every time. Please, look at Java's `PsiUtil.skipParenthesizedExprUp()`.

Reply:
Done.